changelog.md

```diff
@@ -37,6 +37,10 @@
 - Refactor built-in message storage to enable multiple storage engines as well as
   to streamline the implementation of alternative message storage plugins. 
 - Fix metric description string in vmq_bridge.
+- Enable to define the MQTT protocol version to be used by vmq_bridge (4 or 3).
+- Fix vmq_bridge to correctly use the `try_private` config and use the '131'
+  MQTT protocol version instead of '6' used for MQTT 3.1.0.
```

Comment:
Would adding 132 be possible as well? I believe this is also supported by mosquitto?

Reply:
Done, ready to be reviewed